doc/doxygen/headers/laoperators.h

```diff
@@ -89,6 +89,9 @@
  * encapsulation of individual linear operators into blocked linear
  * operator variants.
  *
+ * The step-20 tutorial program has a detailed usage example of the
```

Comment:
Drats, I was too late to comment on this... We could add step-44 to this list too.

Reply:
https://github.com/dealii/dealii/blob/master/examples/step-44/step-44.cc#L3221-L3342